zerver/management/commands/delete_realm.py

```diff
@@ -28,6 +29,12 @@ def handle(self, *args: Any, **options: str) -> None:
 
         print(f"This realm has {user_count} users and {message_count} messages.\n")
 
+        if settings.CORPORATE_ENABLED:
```

Comment:
I usually do `if settings.BILLING_ENABLED` but this is also fine.

Reply:
Aha -- that makes more sense to me!